### PR TITLE
JBPM-6022: Process admin can't release a claimed user task

### PR DESCRIPTION
**Pull request by rbarriuso**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
ok to test

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListPresenterTest.java [hunk 1]
```diff
@@ -52,4 +56,11 @@ public void testResumeActionCondition() {
         testTaskStatusCondition(getPresenter().getResumeActionCondition(),
                                 TASK_STATUS_SUSPENDED);
     }
+    
+    @Test
+    public void testReleaseActionOwnerCondition() {
```
Comment (jhrcek) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListPresenterTest.java]: This test method name is not very informative. Without knowing what problem this PR fixes I wouldn't be able to tell what this method asserts (or to tell without looking at the source code what got broken). How about renaming the test: `public void adminShouldBeAbleToReleaseTaskOwnedByOthers`

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskListPresenterTest.java [hunk 2]
```diff
@@ -51,4 +54,11 @@ public void testResumeActionCondition() {
         testTaskStatusCondition(getPresenter().getResumeActionCondition(),
                                 TASK_STATUS_SUSPENDED);
     }
+    
+    @Test
+    public void testReleaseActionOwnerCondition() {
```
Comment (jhrcek) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskListPresenterTest.java]: Like in the previous commit: how about renaming the method to `public void userShouldNotBeAbleToReleaseTasksOwnedByOthers`

**Comment by rbarriuso** [comment]
Changed test method names as @jhrcek suggested

**Pull request merged**